test/libsolidity/semanticTests/array/copying/copy_byte_array_to_storage.sol

```diff
@@ -48,6 +48,6 @@ contract C {
 // compileViaYul: also
 // ----
 // f() -> 0xff
-// gas irOptimized: 136355
```

Comment:
Why does it get so much more expensive? Shouldn't it get cheaper?

Reply:
I had a look at the diff but could not really make out where the change comes from.

Reply:
I think this is the reason https://github.com/ethereum/solidity/pull/10768/files#r610482903